src/minimax_server/router.clj

```diff
@@ -0,0 +1,18 @@
+(ns minimax-server.router
+  (:import [java.util.function Function]
+           [scarvill.httpserver.response ResponseBuilder])
+  (:require [clojure.string :as str]))
+
+(defn response-with-body [body]
+  (.build (.setBody (new ResponseBuilder) (byte-array (map byte (str body))))))
+
+(defn get-current-player [request]
+  (.getParameterValue request "current_player"))
+
+(defn get-board [request]
+  (str/split (.getParameterValue request "board") #","))
```

Comment:
I would think you'd want the values to be keywords instead of strings here. That would be a more idiomatic way for your business logic to receive the board.